### ISSUE TITLE
payu requires restart***/atmosphere, even if empty directory

### ISSUE THREAD

**Issue by rmholmes** [marshallward/payu#113]
When restarting access-om2 I get errors saying No such file or directory "....archive/restart***/atmosphere". If I make an empty directory restart***/atmosphere then it works, but that directory is empty so it seems unnecessary?

I guess this doesn't affect many people, but when rsyncing restart folders around, if atmosphere is empty then it will not be created in the destination restart***/ and so payu will throw an error unless this directory is manually created.

**Comment by marshallward** [comment]
Yes, I also noticed this yesterday.  This is due to the changes in OM2, specificially because YATM (unlike MATM) does not have any config files.  We can probably make some conditional check to create restart### for each model.

**Comment by aidanheerdegen** [comment]
No recent examples reported. Will close.
